src/number_theory/diophantine_approximation.lean

```diff
@@ -5,16 +5,27 @@ Authors: Michael Geißer, Michael Stoll
 -/
 import tactic.basic
 import data.real.irrational
+import combinatorics.pigeonhole
+import ring_theory.coprime.lemmas
+import data.int.units
+import algebra.continued_fractions.computation.translations
+import algebra.continued_fractions.computation.approximation_corollaries
```

Comment:
I like to sort imports alphabetically, since this makes it clearer which ones are related.
```suggestion
import algebra.continued_fractions.computation.approximation_corollaries
import algebra.continued_fractions.computation.translations
import combinatorics.pigeonhole
import data.int.units
import data.real.irrational
import ring_theory.coprime.lemmas
import tactic.basic
```